tools/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@
 #
 # Distributed under the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+cmake_minimum_required(VERSION 3.7)
```

Comment:
Shouldn't this go into the main CMakeFile.txt?

Reply:
Possibly. I was getting an error message from that file saying it was required.

Reply:
You probably got an error while using this CMakeLists.txt as the main cmake file. Please always use the one in the root directory as your target,